target/linux/ramips/image/Makefile

```diff
@@ -28,6 +28,12 @@ LOADER_FLASH_START := $(ldrflashstart-y)
 
 KERNEL_DTB = kernel-bin | append-dtb | lzma
 
+define Build/humax-factory
```

Comment:
Are you planning to add support for E3 in the near future?

Reply:
No, I don't have E3 and [it's Realtek RTL8197F](https://deviwiki.com/wiki/Humax_Quantum_X3)

Reply:
Oh, I confused E3 with E2...
Then `humax-factory` would be used only by E10 for now, I would move it to mt7621.mk.

Reply:
Relocated!